frontend/src/app/shared/components/file-links/file-link-list-item/file-link-list-item.html

```diff
@@ -1,6 +1,9 @@
-<div class="spot-list--item-floating-wrapper">
+<div
+  class="spot-list--item-floating-wrapper op-files-tab--file-list-item-floating-wrapper"
+  [ngClass]="{ 'no-hover': disabled }"
+>
   <a
-    class="spot-list--item-action op-files-tab--file-list-item-action"
+    class="spot-list--item-action spot-list--item-action_link op-files-tab--file-list-item-action"
```

Comment:
I actually did the same in https://github.com/opf/openproject/pull/10988 😅 I'll close the other one and link the according ticket in the description.